book/display.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+from book.book import Book
+
+
+class Display(ABC):
+    @abstractmethod
+    def display(self, book: Book) -> None:
+        pass
+
+
+class ConsoleDisplay(Display):
```

Comment:
Rename `ConsoleDisplay` to `DisplayConsole`. The logic is that the class name should describe what it does: it displays to the console. The name should read as "Display" (what it does) followed by "Console" (where it does it).